label_studio/core/utils/io.py

```diff
@@ -171,6 +171,7 @@ def __init__(self, iterable):
     def __iter__(self):
         return itertools.chain(self._head, *self[:1])
 
+
```

Comment:
this blank line is extraneous, would favor removing to keep blame clean

Reply:
@jombooth 
I am not sure that this line is extraneous, because it splits class and function outside of the class:

if I remove it, pycharm have a warning:

![image](https://github.com/HumanSignal/label-studio/assets/501780/b081c36b-7d28-48b0-b9b8-6ab0530f885e)